miio/airhumidifier_mjjsq.py

```diff
@@ -0,0 +1,236 @@
+import enum
+import logging
+from collections import defaultdict
+from typing import Any, Dict
+
+import click
+
+from .click_common import EnumType, command, format_output
+from .device import Device, DeviceException
+
+_LOGGER = logging.getLogger(__name__)
+
+MODEL_HUMIDIFIER_MJJSQ = "deerma.humidifier.mjjsq"
+
+AVAILABLE_PROPERTIES = {
+    MODEL_HUMIDIFIER_MJJSQ: [
+        "OnOff_State",
+        "TemperatureValue",
+        "Humidity_Value",
+        "HumiSet_Value",
+        "Humidifier_Gear",
+        "Led_State",
+        "TipSound_State",
+        "waterstatus",
+        "watertankstatus",
+    ]
+}
+
+
+class AirHumidifierMjjsqException(DeviceException):
+    pass
+
+
+class OperationModeMjjsq(enum.Enum):
+    Low = 1
+    Medium = 2
+    High = 3
+    Humidity = 4
+
+
+class AirHumidifierMjjsqStatus:
```

Comment:
How about removing Mjssq from class names now that they are not clashing? The downstreams can do similarly what homeassistant does and import them as whatever they prefer.